test/tests/functional/pbs_cpuset.py

```diff
@@ -125,13 +126,13 @@ def test_reliable_job_startup_on_cpuset(self):
 
         # Check for msg in mom logs
```

Comment:
update the comment to say "check that cpuset has been created for the job"

Reply:
Done